src/models/document/document-content.ts

```diff
@@ -888,6 +898,25 @@ export const DocumentContentModel = types
       }
     }
   }))
+  .actions(self => ({
```

Comment:
Note that this code just moved from further down in the file so I could use it in the copy code.